### DISCUSSION TITLE
Single Field Refresh

### DISCUSSION THREAD

**Discussion by jinu4747** [serenity-is/Serenity#6259] [General]
Hi all,
Is there any way to reload single field instead of loading a page when switching from another tab

Thanks for any help in advance.

**Comment by minhhungit** [comment]
@jinu4747 you can implement an click event on tab element. It will call an endpoint service, the service return data of your field. 
Set the value to form field using:

```typescript
Serenity.EditorUtils.setValue(this.form.YOUR_FIELD_NAME, your_value_here);
```

You can also make a search on this repository for more information about "tabs" https://github.com/serenity-is/Serenity/search?q=this.tabs&type=issues

**Reply by jinu4747** [reply]
@minhhungit Thanks
